timings/helpers.go

```diff
@@ -19,10 +19,21 @@
 
 package timings
 
+import (
+	"fmt"
+)
+
 // Run creates, starts and then stops a nested Span under parent Measurer. The nested
 // Span is passed to the measured function and can used to create further spans.
 func Run(meas Measurer, label, summary string, f func(nestedTiming Measurer)) {
 	nested := meas.StartSpan(label, summary)
 	f(nested)
 	nested.Stop()
 }
+
+// StartupTimestampMsg produce snap startup timings message.
+func StartupTimestampMsg(stage string) string {
```

Comment:
I'm not sure this belongs here until we have a 2nd use case

Reply:
But if someone is going to add a second use case, this file is exactly the place where one would look to find whether such a function already exists :-)

Reply:
the issue is that this package is focused on a very specific concept of timings, now adding this one make it more of a kitchen sync and this new helper is very specific/strange, so for now I would prefer to keep this package focused. The point of a 2nd use case would also understanding how to change this helper, because as it is now this helper hard code a single use case in its name